draft-ietf-jsonpath-base.md

```diff
@@ -1191,7 +1191,7 @@ current-node-identifier = "@"
 Comparison expressions are available for comparisons between primitive
 values (that is, numbers, strings, `true`, `false`, and `null`).
 These can be obtained via literal values; Singular Queries, each of
-which selects at most one node the value of which is then used; and
+which selects at most one node the value of which is then used; or
 function expressions (see {{fnex}}) of type `ValueType` or
 `NodesType` (see {{type-conv}}).
```

Comment:
I think with the addition of `value()`, `NodesType` is no longer valid in a comparison.  Is this an oversight?

Reply:
Good catch. I think this oversight is repeated in 2.6.3: "If it occurs directly as a comparable in a comparison, the function is declared to have a result type of ValueType, or (conversion applies) NodesType." doesn't need to talk about `NodesType`.

Reply:
Addressed in #429.